docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -209,17 +209,7 @@ public class OidcClientResource {
 ----
 ====
 
-If you need you can also create new `OidcClient` without having to configure it in the `application.properties`:
-
-```
-quarkus.oidc-client.client-enabled=false
-
-quarkus.oidc-client.jwt-secret.auth-server-url=${keycloak.url}/realms/quarkus2/
-quarkus.oidc-client.jwt-secret.client-id=quarkus-app
-quarkus.oidc-client.jwt-secret.credentials.jwt.secret=AyM1SysPpbyDfgZld3umj1qzKObwVMkoqQ-EstJQLr_T-1qS0gZH75aKtMN3Yj0iPS4hcgUuTwjAzZr1Z9CAow
-```
-
-Note in this case the default client is disabled with a `client-enabled=false` property. The `jwt-secret` client can be accessed like this:
```

Comment:
I deleted this block because it was repeated some lines before, so I guess it was a copy-paste issue.